pkg/wegoerrors/errors.go

```diff
@@ -0,0 +1,5 @@
+package wegoerrors
+
+import "errors"
+
+var ErrWGEHTTPApiEndpointNotSet = errors.New("the Weave GitOps Enterprise HTTP API endpoint flag (--endpoint) has not been set")
```

Comment:
I suspect you could shorten this to just `ErrWGEEndpointNotSet` and maybe even `ErrNoWGEEndpoint` which would be more in keeping with the Go stdlib https://pkg.go.dev/net/http#pkg-variables and https://pkg.go.dev/io#pkg-variables

Reply:
Oh, and I think this should be in the `cmd` package ?

Because it explicitly refers to a command-line option `--endpoint)` ?

Reply:
Renamed and moved to `cmd` package.